packages/cascara/src/system-components/Boundaries/Boundaries.js

```diff
@@ -1,19 +1,53 @@
-import React from 'react';
+import React, { Suspense } from 'react';
 import pt from 'prop-types';
 import { ErrorBoundary } from 'react-error-boundary';
-import ErrorFallback from './ErrorFallback';
+import { isValidElementType } from 'react-is';
+import { ErrorFallback, SuspenseFallback } from './fallbacks';
+
+// This will need to become a custom prop type we can use elsewhere
+const isReactComponent = (props, propName, componentName) => {
+  if (props[propName] && !isValidElementType(props[propName])) {
+    return new Error(
+      `Invalid prop '${propName}' supplied to '${componentName}': the prop is not a valid React component`
+    );
+  } else {
+    return null;
+  }
+};
```

Comment:
@lordpixel  I created a custom prop-type for checking if this was a component or not. It is a little more safe than just using pt.node.

Reply:
🙌 ok!